about.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Rubik+Dirt&display=swap');
+  </style>
+  <script src="https://kit.fontawesome.com/675259b560.js" crossorigin="anonymous"></script>
+  <script defer type="text/javascript" src="mobileMenu.js"></script>
+  <title>SkyConnect</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar">
+      <div id="upper__nav">
+        <ul class="upper__nav-social ">
+          <li><a href="F" class="nav__elements">
+              <i class="fa-brands fa-facebook-f"></i> </a></li>
+          <li><a href="T" class="nav__elements">
+              <i class="fa-brands fa-twitter"></i>
+            </a></li>
+          <li><a href="English" class="nav__elements">English</a></li>
+          <li><a href="my-page" class="nav__elements">My Page</a></li>
+          <li><a href="logout" class="nav__elements">Logout</a></li>
+        </ul>
+      </div>
```

Comment:
Nice job so far😉. Let's try to make the top nav match the given design. try to add some spacings to the top and bottom of the top dark nav to make it match the design.

yours
<img width="1437" alt="image" src="https://github.com/sinruidotec/Capstone-Project/assets/64643995/832bcacf-d26e-4802-9751-f5aa945c4b50">

design
<img width="886" alt="image" src="https://github.com/sinruidotec/Capstone-Project/assets/64643995/ec98d831-7cce-4cdf-89c1-b7f9ff6e0a34">